### ISSUE TITLE
FPS Limiter Not Working in Steam Performance | ASUS ROG ALLY Z1 EXTREME

### ISSUE THREAD

**Issue by RyzerZ** [ublue-os/bazzite#1733]
### Describe the bug

The FPS limiter in the Steam Performance settings does not function as expected. Despite setting a specific frame rate limit, the FPS continues to exceed this limit during gameplay, leading to performance issues.

### What did you expect to happen?

I expected the FPS to be limited to the specified value, improving overall gameplay performance and stability.

**Comment by matte-schwartz** [comment]
please provide an `rpm-ostree status` output. also, is VRR enabled?

**Comment by RyzerZ** [comment]
After seeing that you mentioned VRR, I tried disabling it, and now the FPS limit works. Were you aware of this issue?

**Comment by matte-schwartz** [comment]
Yes, it is a known issue upstream: https://github.com/ValveSoftware/gamescope/issues/1480

**Comment by RoueGauche303** [comment]
### Similar issue on my Aya Neo Air Pro
For me my display shows up as external display, I have the options for **VRR** & **HDR** but they’re greyed out.

Currently still looking into if there’s a solution, I’m a noob so I do hope I can get some help.

**Comment by EgorAb98** [comment]
https://github.com/ublue-os/bazzite/issues/1733#issuecomment-2406838668

Off vrr and problem still exist. Please help.
